provision/setup-automatic-updates

```diff
@@ -36,9 +38,9 @@ if [ "$VAULT_AUTH_GITHUB_TOKEN" = "" ]; then
     export VAULT_AUTH_GITHUB_TOKEN=${token}
 fi
 vault login -method=github
-SLACK_WEBHOOK=$(vault read -field=password secret/slack/autoupdate-webhook)
+SLACK_WEBHOOK=$(vault read -field=value secret/slack/$CHANNEL)
 
-cat <<EOF > /usr/local/sbin/check_reboot_required.sh
+cat <<EOF > /etc/cron.daily/check_reboot_required
```

Comment:
can you add a bit

```
if [ -f /usr/local/sbin/check_reboot_required.sh ]; then
  rm /usr/local/sbin/check_reboot_required.sh
fi
```

For extra points, you might adapt https://askubuntu.com/a/719877 to remove the cron job, but practically this is not a huge deal as the job will just fail each night

Reply:
cool bananas :banana: